From 30233064aaf5f2492bc687d61d72956876102109 Mon Sep 17 00:00:00 2001
From: Owen O'Malley <omalley@apache.org>
Date: Mon, 24 Sep 2012 03:54:00 +0000
Subject: [PATCH 1344/1344] Fix the length of the secret key.

git-svn-id: https://svn.apache.org/repos/asf/hadoop/common/branches/branch-1@1389197 13f79535-47bb-0310-9956-ffa450edef68
(cherry picked from commit 913afd5f145e957df2381d3de1b90dab885ae856)

Reason: Security update
Ref: CDH-8242
Author: Owen Omalley
---
 .../hadoop/security/token/SecretManager.java       |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/core/org/apache/hadoop/security/token/SecretManager.java b/src/core/org/apache/hadoop/security/token/SecretManager.java
index f5eaf0d..1245e1e 100644
--- a/src/core/org/apache/hadoop/security/token/SecretManager.java
+++ b/src/core/org/apache/hadoop/security/token/SecretManager.java
@@ -75,7 +75,7 @@ public abstract class SecretManager<T extends TokenIdentifier> {
   /**
    * The length of the random keys to use.
    */
-  private static final int KEY_LENGTH = 20;
+  private static final int KEY_LENGTH = 64;
 
   /**
    * A thread local store for the Macs.
-- 
1.7.0.4

